index.html

```diff
@@ -10,10 +11,78 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    
+     
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <header>
+        <nav>
+          <div class="nav-logo">
+            <img class="karma-logo" src="./img/karma-logo.svg" />
+          </div>
+          <div class="nav-logo nav-logo-right">
+            <a class="nav-link active" href="#">Meet Karma</a>
+            <a class="nav-link" href="#">How It Works</a>
+            <a class="nav-link" href="#">Store</a>
+            <a class="nav-link" href="#">Blog</a>
+            <a class="nav-link" href="#">Help</a>
+            <a class="nav-link" href="#">Login</a>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="Karma">
+          <h1 class="Karma-title">Introducing Karma</h1>
+          <h2 class="Karma-description">
+            Bring WiFi with you, everywhere you go.
+          </h2>
+          <a class="btn-Karma" href="#">Learn More</a>
+        </section>
+
+
+        <section class="info">
+              <h2 class="info-title">Everyone needs a little Karma.</h2>
+              <div class="cards-container">
+                  <div class="card">
+                      <img src="img/icon-devices.svg" />
+                      <p>Internet for all devices</p>
+                  </div>
+                  <div class="card">
+                      <img src="img/icon-coffee.svg" />
+                      <p>Boost your productivity</p>
+                  </div>
+                  <div class="card">
+                      <img src="img/icon-refill.svg" />
+                      <p>Pay as You Go</p>
+                  </div>
+          </section>
+      </main>
+
+      <footer class="footer">
+          <p class="footer-join">Join us on</p>
+          <div class="container-social">
+              <div class="social">
+                  <img src="img/twitter-icon.svg" alt="twitter" />
+              </div>
+              <div class="social">
+                  <img src="img/facebook-icon.svg" alt="facebook" />
+              </div>
+              <div class="social">
+                  <img src="img/instagram-icon.svg" alt="instagram" />
+              </div>
+          </div>
+          <p class="footer-text">©Karma Mobility, Inc.</p>
+      </footer>
+    </body>
+  </html>
+    
+
+
+
+
+
 </body>
```

Comment:
these closing `</body>` and `</html>` tags are redundant as you're closing them off above